build/release/Makefile

```diff
@@ -153,7 +153,11 @@ promote.output:
 
 # 1: registry 2: image, 3: arch
 define repo.targets
-build.image.$(1).$(2).$(3): ; @docker tag $(BUILD_REGISTRY)/$(2)-$(3) $(1)/$(2)-$(3):$(VERSION)
+build.image.$(1).$(2).$(3):
+	@docker tag $(BUILD_REGISTRY)/$(2)-$(3) $(1)/$(2)-$(3):$(VERSION)
+	@# Save image as _output/images/linux_<arch>/<image>.tar.gz (no builds for darwin or windows)
+	@mkdir -p $(OUTPUT_DIR)/images/linux_$(3)
+	@docker save $(BUILD_REGISTRY)/$(2)-$(3) | gzip -c > $(OUTPUT_DIR)/images/linux_$(3)/$(2).tar.gz
```

Comment:
might be better to change IMAGE_ARCHS above to not filter out the leading `linux_` and change the docker save here to just use `$(OUTPUT_DIR)/images/$(3)/$(2).tar.gz`

Reply:
I had considered that. As long as it won't cause issues elsewhere, I'll get on it.